src/content/docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure.mdx

```diff
@@ -34,36 +34,252 @@ import kubernetesCke from 'images/kubernetes_logo_cke.png'
 
-## Kubernetes guided install [#guided-install]
+Before you start the guided install, make sure you have a New Relic account. It's free and no credit card is required.
 
-Our guided install helps you with many types of installations: servers, virtual machines, and unprivileged environments. It can also help you with installations in managed services or platforms.
+If your New Relic account [is in the EU region](/docs/using-new-relic/welcome-new-relic/get-started/our-eu-us-region-data-centers), choose the EU install option.
+
+<ButtonGroup>
+  <ButtonLink
+    data-tessen="stitchedPathLinkClick"
+    role="button"
+    to="https://newrelic.com/signup"
+    variant="primary"
+  >
+    Get an account
+  </ButtonLink>
+
+  <ButtonLink
+    role="button"
+    to="https://one.newrelic.com/launcher/k8s-cluster-explorer-nerdlet.cluster-explorer-launcher?pane=eyJuZXJkbGV0SWQiOiJpbnN0YWxsLW5ld3JlbGljLmhvbWUiLCJwYXRoIjoiZ3VpZGVkIiwiZW52Ijoia3ViZXJuZXRlcyIsImFjdGl2ZUVudmlyb25tZW50IjogImt1YmVybmV0ZXMifQ=="
+    variant="primary"
+  >
+    Guided install
+  </ButtonLink>
+</ButtonGroup>
+
+If custom manifests have been used instead of Helm, you will need to first remove the old installation using `kubectl delete -f previous-manifest-file.yml`, and then proceed through the guided installer again. This will generate an updated set of manifests that can be deployed using `kubectl apply -f manifest-file.yml`.
+
+
+### Navigating the Kubernetes Integration guided install [#kubernetes-install-navigation] 
 
 <Callout variant="important">
-Make sure you are using the supported Kubernetes versions. Please check our [compatibility and requirements page](/docs/kubernetes-pixie/kubernetes-integration/get-started/kubernetes-integration-compatibility-requirements).
+Make sure you are using the supported Kubernetes versions and make sure to check out the preliminary notes for your managed services or platforms on our [compatibility and requirements page](/docs/kubernetes-pixie/kubernetes-integration/get-started/kubernetes-integration-compatibility-requirements).
 </Callout>
 
+#### Configure the Kubernetes Integration [#configure-kubernetes-integration] 
+
+**We'll send your data to this account**
+
+Choose the New Relic account that you want your Kubernetes data written to. 
+
+**Cluster name** 
+
+Cluster name is the name we will use to tag your Kubernetes data with so that you can filter for the data specific to the cluster you’re installing this integration in. This is important if you choose to connect multiple clusters to your New Relic account so choose a name that you’ll recognize! 
+
+**Namespace for the integration**
+
+Namespace for the integration is the namespace we will use to house the Kubernetes integration in your cluster. We recommend using the default namespace of ‘newrelic’. 
+
+
+#### Select the additional data you want to gather [#selecting-additional-data-kubernetes-integration] 
+
+##### Scrape Prometheus Endpoints [#scraping-prometheus-endpoints] 
```

Comment:
These h5s are pretty confusing to follow. Hard to tell if they're a subset of 'select additional data' header. Also, there's a couple h6s below; I assume these are subsets of the h5s but it's hard to say. I wonder if we might make a collapser or table. I find a lot of this organization hard to parse

Reply:
@zuluecho9 I had removed all these h5s locally. Now that I've pushed this up, please see the latest commit.